src/Cards/Cards.js

```diff
@@ -20,7 +20,7 @@ export default class Cards extends Component {
   injectMovies = () => {
     return this.state.movies.map(movie => {
       return (
-      <div key={movie.id} className="card">
+      <div key={movie.id} id={movie.id} className="card">
```

Comment:
and ID needs to be given to the movies so we can target the right movie info for iterationn 3